configs/global.toml

```diff
@@ -1,42 +1,42 @@
 client_mode = "MONITOR"
-# blacklist_regex = "^(?:/Users)/.*"
-# whitelist_regex = "^(?:/Users)/.*"
+# blocklist_regex = "^(?:/Users)/.*"
+# allowlist_regex = "^(?:/Users)/.*"
 batch_size = 100
 enable_bundles = false
-enabled_transitive_whitelisting = true
+enabled_transitive_allowlisting = true
 
 [[rules]]
 rule_type = "BINARY"
-policy = "BLACKLIST"
+policy = "blocklist"
```

Comment:
Policy entries should return all caps [per the docs](https://github.com/pmarkowsky/santa/blob/main/docs/development/sync-protocol.md#rules-objects)?

Reply:
Yup. This was an accidental squash issue. I'll fix all of the errors.